src/serialization/sb2.js

```diff
@@ -1061,7 +1061,8 @@ const parseBlock = function (sb2block, addBroadcastMsg, getVariableId, extension
             children: [],
             mutation: {
                 tagName: 'mutation',
-                proccode: procData[0], // e.g., "abc %n %b %s"
+                proccode: procData[0].replace(/\\%/g, '％').replace(/\\(.)/g, '$1'),
```

Comment:
Please create a new function with a comment explaining that these symbols are escaped in scratch 2 project jsons and need to be unescaped when importing into 3.0. 

I think it would be best to use and credit [the `unescape` function in the original scratch-flash code](https://github.com/LLK/scratch-flash/blob/74ae0cbc0427d6acc0508e41a73a324f84ebc8c4/src/util/ReadStream.as#L116-L128) as the basis for this change.